### ISSUE TITLE
lots of thing broken when running from tty instead of gdm

### ISSUE THREAD

**Issue by Brad1944** [hyprwm/Hyprland#7225]
### Regression?

No

### System Info and Version

<details> <summary>System/Version info</summary>
[sysinfo.log](https://github.com/user-attachments/files/16537956/sysinfo.log)
</details>

### Description

Hyprland feels so much different when it's not being ran from gdm

### How to reproduce

Run hyprland via tty or lemurs dm

### Crash reports, logs, images, videos

ags dialog shows weird gray border (?)
![img](https://github.com/user-attachments/assets/42671c65-8125-4e5b-813f-dae29abff735)

window transparency made the whole window invisible
https://github.com/user-attachments/assets/a79cce01-4fa5-4be0-aa83-3653f590eb9e

scrolling feels somewhat slower / different

**Comment by Brad1944** [comment]
[hyprland.log](https://github.com/user-attachments/files/16538714/hyprland.log)
forgot to attach logs

**Comment by LDAP** [comment]
This might be due to the tty shell setting up your environment. Compare the environment variables maybe you find a theme setting in one and not the other?

**Comment by Brad1944** [comment]
> This might be due to the tty shell setting up your environment. Compare the environment variables maybe you find a theme setting in one and not the other?

How can I do so?

**Comment by Brad1944** [comment]
> You start Hyprland and use the printenv command on the terminal (once for tty and once for gdm).

Like this?

[with-gdm.txt](https://github.com/user-attachments/files/16568853/with-gdm.txt)
[with-tty.txt](https://github.com/user-attachments/files/16568854/with-tty.txt)

**Comment by LDAP** [comment]
Yes! However, I didn't find anything out of the ordinary, sadly :(

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
